ETT-Backend.Tests/Services/EmployeeServiceTest.cs

```diff
@@ -1,24 +1,57 @@
+using System.Collections.Generic;
 using Xunit;
+using ETT_Backend.Repository;
+using ETT_Backend.Models;
+using ETT_Backend.Tests.Utils;
 
 namespace ETT_Backend.Services.Test
 {
   public class EmployeeServiceTest
   {
     [Fact]
-    public void RetrieveEmployeeHoursvalidEmail() 
+    public void RetrieveEmployeeHoursValidEmail()
     {
-      var empService = new EmployeeService();
-      var hoursResponse = empService.RetrieveEmployeeMetrics("cmason@callibrity.com");
-      Assert.Equal("0919", hoursResponse.EmployeeId);
-      Assert.Equal(20, hoursResponse.Billable.CurrentHours);
+      List<Employee> dbRetVal = new List<Employee>()
+      {
+        new Employee("lol1", "lol2", "lol3", 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0),
+        new Employee("lul", "lul", "lul", 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0)
+      };
+
+      MockServiceProvider serviceProvider = new MockServiceProvider()
+        .AddMockDBConnection();
```

Comment:
Is the MockDBConnection used since the whole service is mocked?

Reply:
Correct, I created that helper class that can host all of our services in the app service collection (like singleton, scope, stuff in startup.cs).  You can see how I used it in the test.  Basically, I created a setup where if ExecuteQuery is called with a specific query, it returns a specific value.

Reply:
You can add our services to the MockServiceProvider as well to test controllers with.  I can help you with that.